src/login/login.js

```diff
@@ -0,0 +1,25 @@
+import log from 'apify-shared/log';
+import {checkParamOrThrow} from 'apify-client/build/utils';
+import {getLoginFields} from './tools';
```

Comment:
Since this is the only exported function from tools, it should be on the top of that file.

Reply:
Done